build-packages/create-magento-app/index.js

```diff
@@ -138,10 +139,13 @@ yargs.command(
 
         const pathArr = destination.split(path.sep);
         const name = pathArr.slice(-1);
+        const timeStamp = new Date().getTime() / 1000;
 
         await init({
             name, // we do not care about organization it is or not
             path: destination
         });
+
+        googleAnalytics.trackTiming('CMA installation time', new Date().getTime() / 1000 - timeStamp);
```

Comment:
Tracking installation inside the `create-magento-app` package is kinda pointless since it's just running an `npm i` or `yarn` command.

Reply:
Yea, seems so. But maybe you can suggest where we can set tracking for CMA installation in this case ?

Reply:
Looks like you already tracking installation in `magento-scripts` start command :thinking: